packages/jest-circus/README.md

```diff
@@ -1,3 +1,5 @@
+[type-definitions]: https://github.com/facebook/jest/tree/master/packages/jest-types/src/Circus.ts
```

Comment:
```suggestion
[type-definitions]: https://github.com/facebook/jest/blob/master/packages/jest-types/src/Circus.ts
```

Good call to use the cleaner reference style linking 👍  Just a minor detail I spotted; since the link is for a specific object, I believe this should be 'blob' and not 'tree'. If you visit the link you'll see that GitHub redirects

Reply:
Ok, I changed it to `blob`, I believe that perma-links it